### ISSUE TITLE
How to render a new viewpoint based on colmap dense reconstruction?

### ISSUE THREAD

**Issue by dunbar12138** [colmap/colmap#1095]
Is it possible to render a new viewpoint based on colmap dense reconstruction?

**Comment by ahojnnes** [comment]
No, there is no such functionality built into COLMAP. Consider using other approaches, including recent approaches like NeRF, etc.

**Comment by letatanu** [comment]
You can get all 3D points from the dense reconstruction then reproject them to the new viewpoint with the camera intrinsic and extrinsic.

**Comment by prasadvagdargi** [comment]
I would suggest using https://github.com/SBCV/Blender-Addon-Photogrammetry-Importer to import the reconstructed model, after which you can use a new Blender camera to render the scene from that viewpoint. You can set camera intrinsics and extrinsics in Blender.
